modules/os-datagrid-online-services-launch/added/datagrid-online-services.sh

```diff
@@ -6,6 +6,13 @@ CONFIG_FILE=${CONFIG_FILE:=cloud.xml}
 JGROUPS_STACK=kubernetes
 LOGGING_FILE=$JBOSS_HOME/standalone/configuration/logging.properties
 
+if [ "x$JAVA_OPTS" == "x" ]; then
```

Comment:
Nitpick, but would this not better as `if [ -z "$VAR" ];`

Reply:
ooh yeah.. good point. Let me fix it.